test_dir/test_file.md

```diff
@@ -4,6 +4,4 @@ This dir and file created by collaborator user.
 Changes pushed to local fork.
 Later Pull Request should be opened.
 Pull request should be reviewed and approved.
-
-
-This file commented and reviewer askedn to change something.
\ No newline at end of file
+This file commented and the reviewer asked to change something.
```

Comment:
Please add here another line with text "This is another line I was asked to add"